### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
- Added popup HTML
- Added CSS
- Added popup images
- Added popup variables
- Added popup Event listeners and functions
- Added Loop for various popups

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
...
+    <div class="popup hidden">
+      <!-- title and the cancel button-->
+      <div class="popup-heading">
+        <h2>Multi Post Stories</h2>
+        <img
+          class="popup-close"
+          src="images/popup/close.png"
+          alt="close icon"
+        />
+      </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins)

Your Implementation | Figma Design
:---:|:---:
image | image

## Mobile version:
Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 2]
```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
...
+
+        <div class="popup-desc">
+          <p class="popup-text">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
```
Comment (micheaol) [inline index.html]: Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). Please remove all the hard-code from your `html file`

File: index.html [hunk 3]
```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
...
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Pop Up Window -->
+    <div class="popup hidden">
+      <!-- title and the cancel button-->
+      <div class="popup-heading">
+        <h2>Multi Post Stories</h2>
```
Comment (micheaol) [inline index.html]: Please ensure that your Modal popup data is populated with the data from the projects array according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ramrod433 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Mre55 — APPROVED** [review]
**Hi @ramrod433,**

# Status: APPROVED  🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**